bin/patches/@wordpress__data@10.0.2.patch

```diff
@@ -0,0 +1,428 @@
+  *
+- * @param {string}    selectorName Name of selector for which resolver should be invalidated.
+- * @param {unknown[]} args         Arguments to associate for uniqueness.
+- *
+- * @return {{ type: 'INVALIDATE_RESOLUTION', selectorName: string, args: any[] }} Action object.
++ * @param selectorName Name of selector for which resolver should be invalidated.
++ * @param args         Arguments to associate for uniqueness.
+  */
+-export function invalidateResolution(selectorName: string, args: unknown[]): {
++export declare function invalidateResolution<Selectors extends MapOf<Selector>>(selectorName: keyof Selectors, args: SelectorParameters<Selectors[typeof selectorName]>): {
+     type: 'INVALIDATE_RESOLUTION';
+-    selectorName: string;
+-    args: any[];
++    selectorName: keyof Selectors;
++    args: SelectorParameters<Selectors[typeof selectorName]>;
+ };
+ /**
+  * Returns an action object used in signalling that the resolution
+  * should be invalidated.
+- *
+- * @return {{ type: 'INVALIDATE_RESOLUTION_FOR_STORE' }} Action object.
+  */
+-export function invalidateResolutionForStore(): {
++export declare function invalidateResolutionForStore(): {
+     type: 'INVALIDATE_RESOLUTION_FOR_STORE';
+ };
+ /**
+  * Returns an action object used in signalling that the resolution cache for a
+  * given selectorName should be invalidated.
+  *
+- * @param {string} selectorName Name of selector for which all resolvers should
+- *                              be invalidated.
+- *
+- * @return  {{ type: 'INVALIDATE_RESOLUTION_FOR_STORE_SELECTOR', selectorName: string }} Action object.
++ * @param selectorName Name of selector for which all resolvers should be invalidated.
+  */
+-export function invalidateResolutionForStoreSelector(selectorName: string): {
++export declare function invalidateResolutionForStoreSelector<Selectors extends MapOf<Selector>>(selectorName: keyof Selectors): {
+     type: 'INVALIDATE_RESOLUTION_FOR_STORE_SELECTOR';
+-    selectorName: string;
++    selectorName: keyof Selectors;
+ };
+ //# sourceMappingURL=actions.d.ts.map
+\ No newline at end of file
+diff --git a/build-types/redux-store/metadata/selectors.d.ts b/build-types/redux-store/metadata/selectors.d.ts
+index 3a44c74e8bb021444a3072331b546d1b16f4d58e..41bb4f3a712c9c20dafb03ed3750960f3cd1f1e5 100644
+--- a/build-types/redux-store/metadata/selectors.d.ts
++++ b/build-types/redux-store/metadata/selectors.d.ts
+@@ -14,7 +14,7 @@
+  *
+  * @return {StateValue|undefined} isResolving value.
+  */
+-export function getResolutionState(state: State, selectorName: string, args: unknown[] | null): StateValue | undefined;
++export declare function getResolutionState<Selectors extends Record<string, (...args: any[]) => any>>(state: Record<keyof Selectors, State>, selectorName: keyof Selectors, args: SelectorParameters<Selectors[typeof selectorName]> | undefined): StateValue | undefined;
```

Comment:
This changed `null` to `undefined` for `args`. Was the change from null to undefined intentional?

The code seems to accept both `null` and `undefined` for `args`, but JSDoc only mentions `null`. I think we should include `null` for the type. 🤔

Reply:
Other resolution selector functions (getResolutionState, getIsResolving, hasStartedResolution...) also changed from `null` to using `undefined`.

Reply:
No it wasn't intentional, overlooked it. Thanks for picking up on it!